docs/portal/dashboard.md

```diff
@@ -1,9 +1,26 @@
-# Understanding the Dashboard
+# Dashboard
 
-The Dashboard appears when you first log in to the {{title}} portal.
+HDT Dashboard give you information about the traffic of your HDT transports.
 
-Its charts provide a snapshot of your account's traffic. Clicking **View Full Report** in the upper right corner of the chart displays the chart on the [Reports page](</docs/portal/reports.md>), where you can define report parameters, and then view the results on the selected chart.
+If you are a reseller with child accounts, you can select the accounts that the dashboards cover (the parent account only, the child account only, or both the parent and child accounts).
 
-![null](</docs/resources/images/dashboard.png>)
+## Understanding the Dashboard
+
+The Dashboard appears when you first log into the {{title}} portal. The charts provide a snapshot of your account's traffic, covering "Traffic Volume", "Traffic Bandwidth" and "Requests".
```

Comment:
Please consider the following "The Dashboard appears when you first log into the {{title}} portal. Charts provide a snapshot of your account traffic and cover "Traffic Volume", "Traffic Bandwidth" and "Requests".

Reply:
OK, will make the change.